packages/blade/src/components/Input/OTPInput/OTPInput.tsx

```diff
@@ -419,7 +419,7 @@ const _OTPInput: React.ForwardRefRenderFunction<HTMLInputElement[], OTPInputProp
   );
 };
 
-const OTPInput = React.forwardRef<HTMLInputElement[], OTPInputProps>(_OTPInput);
+const OTPInput = memo(React.forwardRef<HTMLInputElement[], OTPInputProps>(_OTPInput));
```

Comment:
What is the cause of re-renders though? Shouldn't it not re-render independent of whether we have memo or not?